release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -1050,6 +1050,11 @@ The Special Resource Operator (SRO) includes a consistent log output format with
 
 Before this update, SRO did not support connecting to registries in disconnected environments. This release provides support for disconnected environments for which the driver container is hosted in a registry outside of the {product-title} cluster.
 
+[id="ocp-4-11-talm-upgrade-recovery"]
+====  Recovery of clusters after a failed upgrade
+
+For {sno}, you can use the {cgu-operator-first} operator to create a backup of a current deployment before an {product-title} version upgrade. If the upgrade fails, you can use the backup to return the cluster to its pre-upgrade state.
```

Comment:
Operator should be capitalized. See Operator in "O" section of SSG https://redhat-documentation.github.io/supplementary-style-guide/#_o

Reply:
See above comment about upgrade/update.